src/main/flight/pid.h

```diff
@@ -48,6 +52,15 @@
 #define ITERM_RELAX_SETPOINT_THRESHOLD 40.0f
 #define ITERM_RELAX_CUTOFF_DEFAULT 20
 
+#define PID_ROLL_DEFAULT  42, 85, 35, 90
```

Comment:
This form for a define is slightly dangerous, as it does not have a guard around it - a better way to do it is to `#define PID_ROLL_DEFAULT { 42, 85, 35, 90 }`.

Reply:
Fixed